examples/src/chains/llm_chain_stream.ts

```diff
@@ -5,7 +5,7 @@ import { CallbackManager } from "langchain/callbacks";
 
 export const run = async () => {
   const manager = CallbackManager.fromHandlers({
-    async handleLLMNewToken(token) {
+    async handleLLMNewToken(token: string) {
```

Comment:
This type is inferred by Typescript, so not terribly useful?

Reply:
my linter always had it red and it was annoying me shrug.

Reply:
oh strange, lets keep your change then